PWGCF/TwoParticleCorrelations/Tasks/CorrSparse.cxx

```diff
@@ -33,12 +37,8 @@ struct CorrSparse {
   Configurable<float> cfgPtCutMin = {"minpt", 0.2, "Minimum accepted track pT. Default 0.2 GeV"};
```

Comment:
For this one, for example, you have to change it to
```
Configurable<float> cfgPtCutMin = {"cfgPtCutMin", 0.2, "Minimum accepted track pT. Default 0.2 GeV"};
```
to get rid of the error in that line
Similar for the similar errors